### PR TITLE
Works section complete

### PR DESCRIPTION
**Pull request by ometman**
I updated the index.html to include a works section. I added portfolio cards about projects and created link buttons access. I added project type, year and brief on each. A total of 4 cards were created based on figma design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,5 +60,149 @@ <h1 class="primary-text">
...
+                        <p class="pr-description">
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+
+                        <div class="pr-button-tag-container">
+                            <button name="html" type="button" class="pr-button-tag"> html </button>
+                            <button name="css" type="button" class="pr-button-tag"> css </button>
+                            <button name="javascript" class="pr-button-tag"> javascript </button>
+                        </div>
```
Comment (Yazino12) [inline index.html:87]: - [ ] Kindly wrap your list of project technologies in a `ul` and each of them a `li` and style them accordingly 👍🏽. You end up with something like this 👇🏽

image

File: index.html [hunk 2]
```diff
@@ -60,5 +60,149 @@ <h1 class="primary-text">
...
+                    <li><a href="#"><img class="social-icon" src="images/medium-icon.png" alt="medium icon"></a></li>   
+                    <li><a href="#"><img class="social-icon" src="images/github-icon.png" alt="github icon"></a></li>    
+                    <li><a href="#"><img class="social-icon" src="images/vic-icon.png" alt="vic icon"></a></li>    
+                </ul> -->
+               <!--  <button name="action" type="button" class="call-to-action"> Get Resume </button>
+                <p class="languages">Languages</p> 
+                <img src="" alt="" class="language-menu"> -->
+                
+
+           <!--  </section> -->
```
Comment (Yazino12) [inline index.html:206]: - [ ] I see that you have commented out your About section entirely 🤔, but remember that it is a requirement for this milestone so please comment it out and style according to the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @ometman 🖐🏾,

You've done well implementing some of the project requirements 🎉🎊🎉, but there are still some you need to work on to move to the next project 🔨.

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 

# STATUS:  REQUIRED CHANGES ♻♻♻ 
TRY AGAIN
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ometman) [reply index.html:206]: Sorry, that was developed in another branch, I kept it from view.  Thanks for the review

**Pull request merged**